packages/amplication-client/src/Settings/ApiTokenList.tsx

```diff
@@ -33,10 +34,19 @@ export const ApiTokenList = React.memo(() => {
     (token: models.ApiToken) => {
       setNewToken(token);
       setNewTokenState(false);
+      setTokenCopied(false);
     },
     [setNewToken, setNewTokenState]
   );
 
+  const handleCopyToken = async () => {
+    const token = data?.userApiTokens[0].token;
+    if (token) {
```

Comment:
For my knowledge of the principles of clean code, you better check if the token doesn't exist and if not, return this function of log some error, not wrap all the code in an if statement

Reply:
@yuval-hazaz do you want to what to this or merge the pr without that?

Reply:
Hi @tupe12334 do you mean something like this:
```
const token = data?.userApiTokens[0]?.token;
if (!token) return console.log("Token does not exist");
await navigator.clipboard.writeText(token);
setTokenCopied(true);
```
Kindly put me through if I am wrong

Reply:
```const token = data?.userApiTokens[0]?.token;
if (!token) {
console.log("Token does not exist");
return
}
await navigator.clipboard.writeText(token);
setTokenCopied(true);```

Reply:
Hi @molaycule more like this, it let you encapsulate the "unhappy" path and continue on the "happy" path

Reply:
Hi @tupe12334 thank you for the feedbacks, I appreciate the review. Kindly check the PR #2604 to resolve this conversation.